chemicaltoolbox/autodock_vina/prepare_box/prepare_box.xml

```diff
@@ -0,0 +1,37 @@
+<tool id="prepare_box" name="Calculate the box parameters for an AutoDock Vina job from an input mol file (confounding box)" version="0.1.0">
+    <requirements>
+        <requirement type="package">rdkit</requirement>
+        <requirement type="package">numpy</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        python $__tool_directory__/calc_vina_box_params.py --ligand '$input1' --config '$output1' --bufx 1 --bufy 2 --bufz 3
+    ]]></command>
+    <inputs>
+        <param type="data" name="input1" format="mol" />
+    </inputs>
+    <outputs>
+        <data name="output1" format="txt" />
+    </outputs>
+    <tests>
+        <test>
+            <param name="input1" value="NUDT5A-x0114_2.mol"/>
+            <output name="output1" file="box_params.txt"/>
+        </test>
+    </tests>
+    <help><![CDATA[
+        usage: calc_vina_box_params.py [-h] [--ligand LIGAND_PATH] [--config OUTPUT]
```

Comment:
Please add some text that indicates what this tool is doing, this will help our users.

Reply:
partially resolved by: f6b92ec865f2975a6772b66ba5eb943f54057122

Will regenerate the xml file when I've finished editing code.

Reply:
It's a good idea to write a help section which is tailored for the graphical interface though, rather than just using one from argparse ([example](https://github.com/bgruening/galaxytools/blob/master/chemicaltoolbox/openbabel/ob_convert.xml)) `planemo tool_init` is a good start but the tool generally needs some further editing.

Reply:
probably resolved by c0d9ecf83a9a57b4011d4e47769ed950fd5ee798